package.json

```diff
@@ -6,12 +6,12 @@
   "scripts": {
     "build": "run-s build:*",
     "build:clean": "if test -d ./lib; then rm -r ./lib; fi",
-    "build:babel": "babel src/ --out-dir lib/ --minified",
```

Comment:
minify should be run at the stage of user's module bundle of client js. browserify and webpack support it.